### PR TITLE
Add static tests

### PR DESCRIPTION
**Pull request by KellerFuchs**

### CODE REVIEW

**Comment by KellerFuchs** [comment]
@hashbang/administrators I guess somebody who understands pip needs to tell me why it fails to find `provisor` while ran on Travis.

**Comment by RyanSquared** [comment]
From Travis CI:

```
Processing ./.tox/dist/hashbangctl-0.2.zip
Collecting provisor (from hashbangctl==0.2)
  Could not find a version that satisfies the requirement provisor (from hashbangctl==0.2) (from versions: )
No matching distribution found for provisor (from hashbangctl==0.2)
```

File: bin/hashbangctl [hunk 1]
```diff
@@ -109,13 +112,13 @@ while True:
 
     key = getch()
 
-    if key in [ key for k,v in menu_items ]:
+    if key in [ key for k, v in menu_items ]:
         print("%s\n" % key)
 
-    if ord(key) == 3: # <Ctrl+C>
-        break;
+    if ord(key) == 3:  # <Ctrl+C>
```
Comment (RyanSquared) [inline bin/hashbangctl:118]: Why the double space?

**Comment by KellerFuchs** [comment]
@ChickenNuggers Yes, I was able to read the log output.
The reason _why_ it fails, though, utterly mystifies me, since `pip install git+https://github.com/hashbang/provisor` obviously succeedded

File: bin/hashbangctl (hunk 1 above)
Comment (KellerFuchs) [inline bin/hashbangctl:118]: @ChickenNuggers It's a PEP8 recommendation, and I agree it improves readability.

**Comment by lrvick** [comment]
This is anchient, and the go-rewrite is nearly done so closing this.

Sorry @KellerFuchs , also we miss you.

**Pull request closed without merging**